src/cartesian/XAxis.tsx

```diff
@@ -30,9 +31,7 @@ interface XAxisProps extends BaseAxisProps {
 
 export type Props = PresentationAttributes<SVGElement> & XAxisProps;
 
-function XAxis(props: Props): any {
-  return null;
-}
+const XAxis: FunctionComponent<Props> = () => null;
```

Comment:
`function Component(props: Props): any`  is unsafe, and also introduces an unused variable, `props` which emits linter warnings.

The change to use the native `FunctionComponent` type is safer, since its return type is not defined as `any`, but rather all the types that are valid as a React component.